target/linux/lantiq/dts/RS353.dts

```diff
@@ -0,0 +1,318 @@
+/dts-v1/;
+
+/include/ "vr9.dtsi"
+
+/ {
+	model = "RS353 - Bintec Elmeg RS353";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200 init=/etc/preinit";
+		leds {
+                        running = &status;
+                        boot = &status;
+			wifi = &wlan;
+			usb2 = &usb; /* Front port is mapped to device 2 */
+			dsl = &dsl;
+                };
+
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	sram@1F000000 {
+		cgu0: cgu@103000 {
+                        compatible = "lantiq,cgu-xway";
+                        reg = <0x103000 0x1000>;
+                        #clock-cells = <1>;
+			lantiq,phy-clk-src = <0x2>;
+		};
+	};
+
+	fpi@10000000 {
+		localbus@0 {
+			nor-boot@0 {
+				compatible = "lantiq,nor";
+				bank-width = <2>;
+				reg = <0 0x0 0x2000000>;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				partitions {
+					compatible = "fixed-partitions";
+					#address-cells = <1>;
+					#size-cells = <1>;
+
+					boardconfig: partition@0 {
```

Comment:
Thanks for the detailed informations. I have to modify comments on my patch, then I'll follow your suggestion, this is my first step into the Linux Kernel contribution ;)

In my side, I wrote some stuff to make the boss-header fully recognised by the OpenWrt build system. The last remaining task is to write a first-boot program that patches the CRC. Regarding the CRC, I can disable the check by setting the length to 0 and crc to 0x00000000. But the resulting image is only flashable by the Bootmonitor via TFTP. 

I commited my code to a new branch [RS353_V2](https://github.com/armSeb/openwrt/tree/RS353_V2) on my GitHub,

As the Other Bintec models has a similar firmware, the  boss-header stuff can be reused for each of them, so porting OpenWrt to a new model is easier ;) For instance the old model RS230 uses a Lantiq Danube SoC, and exactly the same flash layout.